docs/reference/experiment-config.txt

```diff
@@ -208,6 +211,21 @@ created with the Determined CLI. It may contain the following fields:
       and
       :func:`context.get_global_batch_size() <determined.TrialContext.get_global_batch_size>`.
 
+.. _experiment-configuration_training_units:
```

Comment:
Where exactly to put this is a WIP. Thinking it over now.

Reply:
Maybe just a subsection at the bottom of the page?

<img width="1792" alt="Screen Shot 2020-07-28 at 2 52 00 PM" src="https://user-images.githubusercontent.com/9650546/88708600-e798ef00-d0e1-11ea-8649-641bc3ccde23.png">